events/views.py

```diff
@@ -127,3 +134,31 @@ class EventLocationList(ListView):
 
     def get_queryset(self):
         return self.model.objects.filter(calendar__slug=self.kwargs['calendar_slug'])
+
+
+def send_event_mail(creator, event):
+    context = {
+        'event': event,
+        'creator': creator,
+        'site': Site.objects.get_current()
+    }
+
+    text_message_template = loader.get_template('events/email/new_event.txt')
+    text_message = text_message_template.render(context)
+
+    send_mail(
+        subject='Python Events: New event submission',
+        message=text_message,
+        from_email=creator.email,
+        recipient_list=[settings.EVENTS_TO_EMAIL],
+    )
+
+
+class EventSubmit(FormView, LoginRequiredMixin):
```

Comment:
`LoginRequiredMixin` should come before `FormView`.

Reply:
nice catch!